ConsoleUI.js

```diff
@@ -110,6 +166,37 @@ const consoleUI = (function (readline) {
       console.log(
         `Insufficient funds. Player ${player.name} declares bankruptcy.`
       ),
+    mapPropertyShortDisplay,
+    showPlayerTable: (players, properties) => {
+      const propertyText = (prop) => {
+        return `${mapPropertyGroupToColor(prop)} - 💰: $${prop.price}${
+          prop.mortgaged ? ' | ' + c.red('mortgaged') : ''
+        }${prop.buildings > 0 ? ' | ' + prop.buildings + '🏠' : ''}`;
+      };
+
+      let cliTable = new Table();
+      cliTable.push(['', 'position', 'cash', 'assets', 'properties']);
+
+      players.forEach((p) => {
+        cliTable.push([
+          `${p.name} ${p.jailed !== -1 ? '🔒' : ''}${p.bankrurpt ? '💀' : ''}`,
```

Comment:
We can switch `p.jailed !== -1` for `!~p.jailed`, but it requires folk to know `~` so that earns negative readability points

Reply:
updated for fancy pants.